components/chosenArticlePage/VotingContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+
+import Button from 'react-bootstrap/Button';
+import patchArticleVotes from '../../utils/patchArticleVotesByArticleId';
+
+
+export default function VotingContainer ({votes, setVotes, chosenArticleId}) {
+
+    const [updateVotes, setUpdateVotes] = useState([])
+    
+    const handleUpVote =  (e) => {
+        setUpdateVotes({
+            inc_votes : 1
+        })
+        }
+        
+        const handleDownVote = (e) => {
+            setUpdateVotes({
+                inc_votes : -1
+            })
+        }
+        
+        
+        useEffect(() => {
+            setVotes(votes)
+        },[votes])
+
+        useEffect(() => {
+             patchArticleVotes({votes, setVotes, chosenArticleId, updateVotes}) 
```

Comment:
Great that you've separated out this api call into a separate file, keeps the code tidy! 🧹